src/size-rhythms.md

```diff
@@ -0,0 +1,65 @@
+
+Even though 6 is not a multiple of 4, it sums up to a 12 units distance between two buttons in a row; and since 12 is a multiple of 4, the result will be moa👌.
+
+<limel-example-size></limel-example-size>
+
+
+## REM or PX
+We only use `rem` units in our designs system, based on browsers' default base unit of measurement (where `16px` = `1rem`). In such a system, multiples of 4px (our core size unit) can easily be calculated, as they increase sequentially by `0.25`.
+
+* 24 px = 1.5 rem
+* 20 px = 1.25 rem
+* 16 px = 1 rem
+* 8 px = 0.5 rem
+* 4 px = 0.25 rem
+
+However, while designing a UI, it may be easier to think in pixels, as rem is more abstract. By using a custom SASS function like below, you can effortlessly leave the job of unit conversion to the computer.
+
+````css
+/// first define this converter function in your SCSS ···
+@function pxToRem($px) {
+    @return #{$px/16}rem;
+}
+
+// ··· and then use it to ease calculations
+.example {
+    width: pxToRem(4); // will be compiled to 0.25rem
+}
+````
+
+If you are developing new components for Lime Elements, you should know that this function is already available and can be easily imported into your `.scss` files via: `@import '../../style/mixins';`
+
+So if you think something should be rendered 40px wide on the screen, you you don't need to calculate the respective value in REM. Just type `width: pxToRem(40);` in your SCSS code.
+
+**Note** write `1px` not `0,0625rem`!
+
+Anything that should get rendered in `1px` on a screen (such as a border) should be written in pixels. It's very important not to write it in its rem equivalent (`0,0625rem`). This is because depending on the physical pixel density of screens, a `0,0625rem` thick border for example, may get rendered like a semi-transparent line; or even worse, never get rendered in some situations.
+
+## Exceptions in sizing rhythms
+There are exceptions where we do not have to follow the abovementioned sizing convention (multiples of 4). These exceptions are:
+* Numbers `1` and `2` are allowed to use when defining some visual details such as borders.
+* `font-size`: how fonts are rendered depend on the typography and design of the font itself. Fonts do not follow the same logic of simple geometric shapes of the UI. Therefore, using incremental rhythm based on 4 for font sizes (e.g. for H1, H2, H3, etc...) may generate huge font sizes which won't be usable. Also other numbers such as 14 or 13 may appear more appealing for a balanced and easily readable text, compared to 16, or 12.
+* `border-radius`: how a value of 4 is rendered as a border radius is slightly too-little for the type of visual expression we want to achieve for example on a small clickable element like a button. This is while a value of 8 may be too much. Thus, the designer may pick an arbitrary value that helps the visual expression.
```

Comment:
```suggestion
* `border-radius`: a border radius of 4 is slightly too little for the type of visual expression we want to achieve for example on a small clickable element like a button. This is while a value of 8 may be too much. Thus, the designer may pick an arbitrary value that helps the visual expression.
```

This whole bullet point is a bit rambling, and hard to follow. Can it be simplified?

How about something like this?

```suggestion
* `border-radius`: for small elements, jumps of 4 may be too large for the border-radius. Pick the value that best helps the visual expression, even if it's not a multiple of 4.
```

Reply:
OK, what about this:

* `border-radius`: when it comes to border radius, multiples of 4 may render too large or too little. Pick the value that best helps the visual expression you want to achieve, even if it's not a multiple of 4.

Reply:
👍 with this tiny change:

```suggestion
* `border-radius`: when it comes to border radius, multiples of 4 may render too large or too small. Pick the value that best helps the visual expression you want to achieve, even if it's not a multiple of 4.
```

("small" instead of "little")